community-membership.md

```diff
@@ -88,6 +88,9 @@ List of contributions:
   - Code is well tested
   - Tests consistently pass
   - Addresses bugs or issues discovered after code is accepted
+- Can do a `/lgtm` on open PRs.
+- Can be assigned to issues and PRs, and people can ask you for reviews with a `/cc @username`.
+- No `/ok-to-test` needed.
```

Comment:
Something like "Tests can be run against their PRs automatically".

Reply:
Members can do a `/ok-to-test` for PRs that have `needs-ok-to-test` and use commands like `/close` to close PRs as well.